ac.soton.xeventb.sdk/feature.properties

```diff
@@ -23,6 +23,15 @@ This feature provides eXtended Event-B (XEvent-B) and Tests together with their
 source and the source code and tests (SDK version).\n\
 -------------------------------------------------------------------\n\
 Release history:\n\
+0.0.7\n\
```

Comment:
@dd4g12 Should these details changes (e.g., on Event-B XText Machine, etc.) be in the Release history for the ac.soton.xeventb.feature. For the SDK, since it is the wrapper, it is probably sufficient to include simple things like new version number for the included features.